### PR TITLE
Feature/hmw 73 add counts to accordion

### PR DESCRIPTION
**Pull request by cschwinderg**
## Related Issues:
* https://jira.epa.gov/browse/HMW-73

## Main Changes:
* Fixed the counts on the Swimming, Eating Fish, Aquatic Life, and Drinking Water tabs.
  * This was caused by c3e2ff6. I ended up reverting part of that commit, since the `field`  variable comes from the web service.
* Added counts to the accordion headers to make the app more consistent.
* Added counts to the sub accordion content on the Protect tab.

## Steps To Test:
1. Navigate to http://localhost:3000/community/040302020702/protect
2. Expand each accordion item, except for the "Watershed Health Scores" item
3. Verify each one has a count, except for the "Watrershed Health Scores"
4. Go through each main tab (i.e. Overview, Swimming, etc.)
5. Verify the accordion headers have counts in them now.

### CODE REVIEW

**Comment by courtneymyers** [comment]
@cschwinderg my bad on c3e2ff6! I didn't realize those properties came from the web service... I should have tested that a bit more carefully

**Comment by cschwinderg** [comment]
@courtneymyers No worries. I missed that as well.

File: app/client/src/components/pages/Community/components/tabs/Protect.js [hunk 1]
```diff
@@ -1202,10 +1223,13 @@ function Protect() {
...
                             <>
-                              <p>
+                              <div css={modifiedInfoBoxStyles}>
+                                <strong>
+                                  {allProtectionProjects.length.toLocaleString()}
+                                </strong>{' '}
                                 EPA funded protection projects in the{' '}
                                 <em>{watershed}</em> watershed.
-                              </p>
+                              </div>
```
Comment (courtneymyers) [inline app/client/src/components/pages/Community/components/tabs/Protect.js:1232]: The wrapping `<div css={modifiedInfoBoxStyles}>` is correct, but there should be a paragraph inside as well... I just realized we should to update two other pages to have the correct structure as well:
- the Community Overview component
- the WaterbodyList component

If you do a search for `<div css={modifiedErrorBoxStyles}>` you'll see those all use the correct nested structure of div > p.

Comment (cschwinderg) [reply app/client/src/components/pages/Community/components/tabs/Protect.js]: Good catch! I went ahead and resolved both of these issues.

Comment (courtneymyers) [reply app/client/src/components/pages/Community/components/tabs/Protect.js]: thanks!

**Review by courtneymyers — APPROVED** [review]

**Pull request merged**